src/app/components/views/login-view/login-view.component.scss

```diff
@@ -0,0 +1,6 @@
+.container {
```

Comment:
instead of changing the bootstrap values of `.container` - create new class and add that with the container class in the html.    if `.container` class is used somewhere else in this view...it will be affect by these changes